auth.go

```diff
@@ -166,6 +167,8 @@ type CentralStats struct {
 	GoodOnceOffAuth    uint64
 	GoodLogin          uint64
 	Logout             uint64
+	UserLoginAttempts  map[string]uint64
+	loginLock          sync.Mutex
```

Comment:
Make the name of this mutex precisely match the variable that it guards, for example,
`userLoginAttemptsLock`, and then it's a good idea also to comment on it "Guards all access to UserLoginAttempts"